lib/ocpp/common/websocket/websocket_tls.cpp

```diff
@@ -200,6 +235,8 @@ tls_context WebsocketTLS::on_tls_init(std::string hostname, websocketpp::connect
         }
 
         context->set_verify_mode(boost::asio::ssl::verify_peer);
+        context->set_verify_callback(websocketpp::lib::bind(
+            &verify_csms_cn, hostname, websocketpp::lib::placeholders::_1, websocketpp::lib::placeholders::_2));
```

Comment:
`clang-tidy` reports here: "Prefer a lambda to std::bind" (see https://clang.llvm.org/extra/clang-tidy/checks/modernize/avoid-bind.html).

Do you have `clang-tidy` activated locally? E.g. with the `clangd` VSCode-extension.